client/src/Pages/Login/LoginView.js

```diff
@@ -26,18 +29,19 @@ const LoginView = props => {
   if (!loading && data && data.login && data.login.successful) {
     props.handleLoggedIn({
       authToken: data.login.authToken,
-      userId: data.login.userId
+      userId: data.login.userId,
     });
   }
 
   return (
     <>
+      {console.log(data, 'this is the data from loginviewq')}
```

Comment:
remove. Can you make sure there is an eslint rule that prevents console.log? (`no-console` rule)